lib/src/multipart_body.dart

```diff
@@ -8,19 +8,17 @@ final _random = Random();
 
 class MultipartBody implements RequestBody {
   final List<Part> parts;
-  MediaType _contentType;
+  MediaType type;
   String _boundary;
 
   MultipartBody({
     this.parts = const [],
+    this.type = MediaType.multipartMixed,
```

Comment:
Should use `multipartFormData` as default value.

```suggestion
    MediaType type = MediaType.multipartFormData,
```